layers/enums.go

```diff
@@ -102,10 +102,10 @@ const (
 	LinkTypeSLIP           LinkType = 8
 	LinkTypePPP            LinkType = 9
 	LinkTypeFDDI           LinkType = 10
+	LinkTypeRaw            LinkType = 12
```

Comment:
I believe it should be fine... LinkType implements a Decoder, but there's nothing against different link types decoding into the same layer type.